content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "About Arduino accounts for minors"
+---
+
+Arduino offers two types of accounts for minors: <a href="#Junior-accounts">Junior accounts</a> and <a href="#Adult-accounts">Adult accounts for users between 14 and 16 years old</a>. They differ in the features that users can access and the way that Arduino handles [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy). Both account types require the minor's account to be linked to a person exercising parental authority. Learn more about Arduino accounts for minors below.
+
+<h2 id="Junior-accounts"></h2>
+
+## Junior accounts
+
+This type of account is intended for minors under the age of 14. Arduino does not process any [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy) that would allow users to be identified as a child under the age of 14.
+
+In order to [create a Junior Account](https://support.arduino.cc/hc/en-us/articles/360022234360-Create-an-Arduino-account-for-juniors#junior-request), the user is asked to choose a username and a picture (an Avatar) from a set of predetermined options, to enter a password of their choice, and to provide their date of birth.
+
+To complete the registration of a Junior Account, it is necessary to provide a parent's email. Arduino will process the [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy) of the person exercising parental authority.
+
+> If the parent's email is not provided the Junior Account will not be created and all data provided will be deleted within 30 days.
+
+**With a Junior Account, users are able to:**
+
+* Access Educational products and services intended for middle school students (view the full list on [Arduino Education](https://www.arduino.cc/education)).
+* Download and use Arduino IDE.
+* Download and use the Arduino Science Journal App. Use the Science Journal Apps only for creating experiments saved locally in the App. Sharing functionality with Google Drive is not made available to Junior Accounts.
+* Submit Assignment from the Science Journal App to Google Classroom, subject to an existing plan held by your teacher and approval of the latter.
+
+### Upgrading to an Adult account
+
+After turning 14, users of Junior accounts will be prompt to upgrade to an <a href="#Adult-accounts">Adult account for users between 14 and 16 years old</a>.
+
+![Upgrading to regular account pop-up](img/Upgrade_to_regular_account.png)
+
+This process will move all their files, projects & data to the upgraded account, but the account will still require a person exercising parental authority.
```

Comment:
```suggestion
This will transfer all their files, projects and data to the upgraded account, but the account will still require a person exercising parental authority.
```